docs/config.rst

```diff
@@ -120,6 +120,15 @@ mitie_file
 :Description:
     File containing ``total_word_feature_extractor.dat`` (see :ref:`section_backends`)
 
+spacy_model_name
+~~~~~~~~~~~~~~~~
+
+:Type: ``str``
+:Examples: ``"en_core_web_sm"``
+:Description:
+    If the spacy model to be used has a name that is different from the language tag (which is e.g. ``"en"``),
```

Comment:
Just some grammar thing, If I remember correctly, you cannot use `e.g.` in the middle of the sentence, but only in parentheses or in an apposition. Here you probably want `... that is different from the language tag (en, de, etc. )`